src/09_dictionaries.py

```diff
@@ -36,12 +36,20 @@
 # Add a new waypoint to the list
 # YOUR CODE HERE
 
+waypoints.append(
+    {"lat": 35, "lon": -133, "name": "Somewhere over the rainbow"})
+print(waypoints)
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
 
 # YOUR CODE HERE
+waypoints[0].update({"lat": 43, "lon": -130, "name": "not a real place"})
+print("NEW DICT", waypoints)
```

Comment:
Ideally this would have looped through waypoints to find the one with the name "a place" and then changed the name and then updated it.